src/sql/src/names.rs

```diff
@@ -1218,6 +1230,7 @@ struct ItemResolutionConfig {
 pub struct NameResolver<'a> {
     catalog: &'a dyn SessionCatalog,
     ctes: BTreeMap<String, LocalId>,
+    continual_task: Option<(PartialItemName, LocalId)>,
```

Comment:
glad to see this worked out!

Reply:
Thanks for all the time hashing it out with me!